app/controllers/clearance/sessions_controller.rb

```diff
@@ -33,4 +37,8 @@ def url_after_create
   def url_after_destroy
     sign_in_url
   end
+
+  def redirect_url
```

Comment:
ok, I was not sure if another method was needed since it was somewhat incorrect about what the url was actually for